src/ints/int10_modes.cpp

```diff
@@ -1647,84 +1706,112 @@ bool INT10_SetVideoMode(uint16_t mode)
 			att_data[0x09]=0x08; // low-intensity in blink-off case
 			att_data[0x0d]=0x18; // high-intensity in blink-off
 			break;
+
 		case 0x11:
 			for (i=1;i<16;i++) att_data[i]=0x3f;
 			break;
+
 		case 0x10:
 		case 0x12:
 			goto att_text16;
+
 		default:
-			if ( CurMode->type == M_LIN4 )
+			if (CurMode->type == M_LIN4) {
 				goto att_text16;
-			for (uint8_t ct=0;ct<8;ct++) {
-				att_data[ct]=ct;
-				att_data[ct+8]=ct+0x10;
+			}
+			for (uint8_t ct = 0; ct < 8; ct++) {
+				att_data[ct]     = ct;
+				att_data[ct + 8] = ct + 0x10;
 			}
 			break;
 		}
 		break;
+
 	case M_TANDY16:
 		// TODO: TANDY_16 seems like an oversight here, as
 		//       this function is supposed to deal with
 		//       MCH_EGA and MCH_VGA only.
-		att_data[0x10]=0x01;		//Color Graphics
-		for (uint8_t ct=0;ct<16;ct++) att_data[ct]=ct;
+
+		// Color Graphics
+		att_data[0x10] = 0x01;
+		for (uint8_t ct = 0; ct < 16; ct++) {
```

Comment:
Strictly speaking, it would have to become the following:
```suggestion
		for (uint8_t ct = 0; ct < 16; ++ct) {
```
But since this is obviously a reformat, this can wait for some other time. 😄